Sources/SwiftGen/Commander/ConfigCommands.swift

```diff
@@ -162,3 +160,20 @@ enum ConfigCLI {
     NSWorkspace.shared.open(docURL)
   }
 }
+
+private extension Config {
+  func runCommands(verbose: Bool) throws {
+    let errors = commands.keys.sorted().parallelCompactMap { cmd -> Swift.Error? in
+      do {
+        _ = try run(cmd: cmd, verbose: verbose)
```

Comment:
Why the need for `_ =`? Doesn't `run(cmd:verbose:)` return `Void`?

Reply:
Yeah, that shouldn't be needed afaik, the signature is:
```swift
func run(cmd: String, verbose: Bool) throws
```

Reply:
i was using `try?` initially, that's why it was there 😅